src/locales/en.json

```
@@ -577,6 +577,7 @@
   "audio-track": {
     "aria-label": "{title} - Audio Player",
     "aria-label-interactive": "{title} - audio player - press the spacebar to play and pause a preview of the audio",
+    "aria-label-interactive-seekable": "{title} - audio player - press the spacebar to play and pause a preview of the audio; use the left and right arrow keys to seek through the waveform",
```

I think the word 'track' is more appropriate here as compared to 'waveform'.

```suggestion
    "aria-label-interactive-seekable": "{title} - audio player - press the spacebar to play and pause a preview of the audio; use the left and right arrow keys to seek through the track",
```

---

Do you think we should also add a note that this is also a link, and if you press Enter, you'll see the details of the audio, for user navigating with VoiceOver? Currently, it might seem like a list of playable audios without links.

---

Ideally the nature of the element being an anchor tag would be pointed out by the screen-reader.

---

Changed in 8508e4f.